src/index.html

```diff
@@ -27,11 +27,15 @@
           </div>
         </li>
         <li class="input-text list-item">
-          <input type="text" placeholder="Add to your list...." />
+          <input
+            type="text"
+            placeholder="Add to your list...."
+            id="description"
```

Comment:
- [ ] validate the form to avoid submit empty field.
![Screenshot from 2022-04-28 14-46-34](https://user-images.githubusercontent.com/65860678/165756357-1b08f2bf-499e-4e13-ba3a-38c38aaf8171.png)